cypress/integration/edit_lay_percentage.spec.js

```diff
@@ -0,0 +1,13 @@
+describe("Edit lay percentage", () => {
+    beforeEach(() => {
+        cy.setCookie("Other", "other");
+        cy.setCookie("XSRF-TOKEN", "abcde");
+        cy.visit("/random-reviews/edit/lay-percentage");
+    });
+
+    it("allows me to change the lay percentage value and throws an error after inputting the incorrect value", () => {
```

Comment:
This still isn't describing what you are testing. "throws an error after inputting the incorrect value" is fine but you aren't changing the lay percentage value.

Reply:
Doh! Done